Assets/MixedRealityToolkit/_Core/Inspectors/PropertyDrawers/InputActionPropertyDrawer.cs

```diff
@@ -19,7 +19,7 @@ public class InputActionPropertyDrawer : PropertyDrawer
 
         public override void OnGUI(Rect rect, SerializedProperty property, GUIContent content)
         {
-            if (!MixedRealityManager.HasActiveProfile)
+            if (!MixedRealityManager.IsInitialized || !MixedRealityManager.HasActiveProfile)
```

Comment:
just use HasActiveProfile, as IsInitialized is checked within HasActiveProfile

Reply:
I tried that but still got a null ref on the MR Manager so I added it back.

Reply:
odd, might need to check that function then

Reply:
I think it's bc the MR Manager is in the process of initializing and this get's raised from a different thread in the editor.